ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,206 @@
     "FieldType": "Document",
     "ElementLabel": "Upload a file",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "telephone",
+    "ListElementCode": "telephoneNumber",
+    "FieldType": "Text",
+    "ElementLabel": "Telephone",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "partyId",
+    "FieldType": "Text",
+    "ElementLabel": "Party ID",
+    "FieldShowCondition": "partyType=\"DO_NOT_SHOW\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "partyType",
+    "FieldType": "Text",
```

Comment:
Wouldn't it be better to choose party type from radio group? Allowing users to type it in free text field sounds like a bad idea to me.

Reply:
I was populating this in the migration as individual. It can only be individual or organisation correct? Happy to change to another type - shall ask Mia and Eilis now

Reply:
I wonder what Mia and Eilis think from user perspective however if we need to have it fixed list of choices is better than free text from data mining perspective.